server/db/models/boats.js

```diff
@@ -0,0 +1,25 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+module.exports = db.define('boats', {
```

Comment:
I think ```const Boat = db.define,```

and then at bottom ```module.exports = Boat```

Similar to the User file / model definition

Reply:
Both way work fine